index.html

```diff
@@ -302,10 +302,10 @@ <h4 class="desktop-only">FACEBOOK</h4>
 
             <img class="escritorio-trabajo-foto-izquierda" src="media/escritoriotrabajo2.png" alt="">
          
-         </div>
+         </main>
```

Comment:
- I recommend that you don't forget to change your </main> tag to </section> to follow good HTML practices.